packages/web-wallet/pages/_app.tsx

```diff
@@ -18,16 +18,13 @@ import {ContactRoute, DataProvider, DataResource, KeyManagementRoute, MainRoute}
 import {keysDataProvider} from '@/src/dataProviders/keysDataProvider'
 import {identifiersDataProvider} from '@/src/dataProviders/identifiersDataProvider'
 import {presentationDefinitionDataProvider} from '@/src/dataProviders/presentationDefinitionDataProvider'
+import {credentialDataProvider} from '@/src/dataProviders/credentialDataProvider'
```

Comment:
import form /src?
Are the exports all ok?

Reply:
i only moved the import up, i think they are ok, but our of scope anyways